### DISCUSSION TITLE
Reviving Adiantoideae?

### DISCUSSION THREAD

**Discussion by choess** [pteridogroup/ppg#53] [Ideas]
I'm curious as to what people think about the merits of reviving Pteridaceae subfam. Adiantoideae, as distinct from Vittarioideae. The adiantoids (s.s., i.e., _Adiantum_) and the vittarioids were certainly distinguished in the pre-molecular era, but if I recall correctly, the two were lumped in PPG I as it was not yet certain that they were reciprocally monophyletic, as has been inferred in more recent studies.

On the one hand, this could be condemned as a somewhat splitty proposal, creating a monogeneric subfamily. On the other hand, the two clades are clearly distinguishable, and more or less morphologically coherent (with the exception of _Rheopteris_ among the vittarioids). It feels like it would be nice to have a ranked name to hold records of vittarioids where micromorphological examination (and identification to genus) is not possible without having to include _Adiantum_.

I'd like to know if anyone else thinks this might be worth writing up as a proposal.

**Comment by crothfels** [comment]
I think this is a good idea! There are lots of cases where someone might colloquially use the term "vittarioid" in the more restricted, non-_Adiantum_ sense (I've used it in a paper title!), and that clade has a long history of recognition at the family rank, so I think it would be useful to have a formal taxon name for this group. (And subfamily is the obvious choice here -- it's also worth mentioning that _Adiantum_, while a single genus, is quite heterogeneous and diverse, including featuring deep divergences among its major subclades, so is comparable to the vittarioids s.str in many ways). Also, subfamilies aren't used much except by those who find them useful for a particular application, so this "split" wouldn't pose much of a burden on users of fern taxonomy. 

As an example of why recognizing both Adiantoideae and Vittarioideae would be helpful, when providing identifications in iNaturalist, or searching the iNaturalist records, one might want to specify the vittarioids (s.str) without also including _Adiantum_.

**Comment by DrAshleyField** [comment]
I think the introduction of Adiantoideae just to make Vittarioideae strict sense available is useful and I recommend a proposal be put up. We already use "vittarioid" as an ID when doing epiphytes surveys as a number of genera and species require up-close examination for identification.

**Comment by MichaelKessler979** [comment]
More generally, what is the general feeling about elevating the Pteridaceae subfamilies to family level? There are good arguiments for keeping Pteridaceae as it is, but I guess we all know how cumbersome that family is. Ever tried to teach students how to recognize the family? I always show them the subfamilies and then explain that (for now?) they are all in one very heterogeneous family. Would a proposal in this direction make sense?

**Comment by gonggashan88** [comment]
Hi Michael @MichaelKessler979, I would vote for splitting Pteridaceae s.l. into families! Back in 2011, when Alan (Smith) and I were discussing fern families for the Flora of China, I was suggesting that we split Pteridaceae into five families without any subfamilies, and Alan was kindly "willing to accept these". For some (political) reason the Flora of China later fully adopted the Christenhusz et al. (2011) arrangement in which Pteridaceae were maintained as broadly defined (Tryon, Tryon & Kramer, 1990; Smith et al., 2006). 

If they are going to be split, should six families, Adiantaceae, Cryptogrammaceae, Hemionitidaceae, Parkeriaceae, Pteridaceae s.s., and Vittariaceae be recognized?
